internal/service/flexcluster/data_source_schema.go

```diff
@@ -0,0 +1,108 @@
+					MarkdownDescription: "Storage capacity available to the flex cluster expressed in gigabytes.",
+				},
+				"provider_name": schema.StringAttribute{
+					Computed:            true,
+					MarkdownDescription: "Human-readable label that identifies the cloud service provider.",
+				},
+				"region_name": schema.StringAttribute{
+					Computed:            true,
+					MarkdownDescription: "Human-readable label that identifies the geographic location of your MongoDB flex cluster. The region you choose can affect network latency for clients accessing your databases. For a complete list of region names, see [AWS](https://docs.atlas.mongodb.com/reference/amazon-aws/#std-label-amazon-aws), [GCP](https://docs.atlas.mongodb.com/reference/google-gcp/), and [Azure](https://docs.atlas.mongodb.com/reference/microsoft-azure/).",
+				},
+			},
+			Computed:            true,
+			MarkdownDescription: "Group of cloud provider settings that configure the provisioned MongoDB flex cluster.",
+		},
+		"tags": schema.MapAttribute{
+			ElementType:         types.StringType,
+			Computed:            true,
+			MarkdownDescription: "Map that contains key-value pairs between 1 to 255 characters in length for tagging and categorizing the instance.",
+		},
+		"backup_settings": schema.SingleNestedAttribute{
+			Attributes: map[string]schema.Attribute{
+				"enabled": schema.BoolAttribute{
+					Computed:            true,
+					MarkdownDescription: "Flag that indicates whether backups are performed for this flex cluster. Backup uses [TODO](TODO) for flex clusters.",
+				},
+			},
+			Computed:            true,
+			MarkdownDescription: "Flex backup configuration",
+		},
+		"cluster_type": schema.StringAttribute{
+			Computed:            true,
+			MarkdownDescription: "Flex cluster topology.",
+		},
+		"connection_strings": schema.SingleNestedAttribute{
+			Attributes: map[string]schema.Attribute{
+				"standard": schema.StringAttribute{
+					Computed:            true,
+					MarkdownDescription: "Public connection string that you can use to connect to this cluster. This connection string uses the mongodb:// protocol.",
+				},
+				"standard_srv": schema.StringAttribute{
+					Computed:            true,
+					MarkdownDescription: "Public connection string that you can use to connect to this flex cluster. This connection string uses the `mongodb+srv://` protocol.",
+				},
+			},
+			Computed:            true,
+			MarkdownDescription: "Collection of Uniform Resource Locators that point to the MongoDB database.",
+		},
+		"create_date": schema.StringAttribute{
+			Computed:            true,
+			MarkdownDescription: "Date and time when MongoDB Cloud created this instance. This parameter expresses its value in ISO 8601 format in UTC.",
+		},
+		"id": schema.StringAttribute{
+			Computed:            true,
+			MarkdownDescription: "Unique 24-hexadecimal digit string that identifies the instance.",
+		},
+		"mongo_dbversion": schema.StringAttribute{
```

Comment:
```suggestion
		"mongo_db_version": schema.StringAttribute{
```
for consistency throughout all resources

Reply:
Fixed! 👍